src/SmallStrainPlasticity/Elasticity.jl

```diff
@@ -0,0 +1,35 @@
+using Tensors
+abstract type AbstractElasticity{T} <:AbstractMaterial end
```

Comment:
I am not a fan of having to many abstract types. In this case, will there ever be more then one elastic version of the material?

Reply:
Yes, we could have e.g. LinearOrthotropicElasticity (or even more general anisotropic materials). Technically also non-linear versions, but this is more natural for the finite strain implementation...